src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentDay = 40;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
+  let result = days * rentDay;
+
+  if (days >= 7) {
+    result = result - discountSevenDays;
+  } else if (days >= 3) {
```

Comment:
try to avode `else / if` contraction
```suggestion
  } 
  
  if (days >= 3) {
```